### ISSUE TITLE
ctrl+shift+e shows question marks during emoji preview

### ISSUE THREAD

**Issue by Riamse** [ibus/ibus#2107]
Please fill in the following items if you don't know the root cause.

Which distribution and version?: Arch Linux

Which desktop environment and version?: Plasma 5.16.3

Which session type?: X11

Which application and version?: kate and keepassXC - this might be a Qt issue.

IBus version?: 1.5.20

Issue description:

When typing an emoji, the user is given a chance to see what emoji they typed before committing to it and actually putting it into the box. I refer to this as the "emoji preview". For some applications, the emoji preview stage only shows question marks instead of the actual emoji:
![image](https://user-images.githubusercontent.com/2367611/61170886-672e3d00-a524-11e9-825e-3ae584e6528f.png)

Steps to reproduce:
1. ctrl+shift+e
1.  type "thinking" then hit space
1. _text box will show: e�� instead of the emoji_ (this is the bug!)
1. hit enter. text box will show correct emoji.

Can you reproduce your problem when you restart ibus-daemon? **yes**

Do you see any errors when you run ibus-daemon with the verbose option?:
(Run `ibus-daemon --xim --verbose &` and look at the output when you encounter your problem.)

Can you reproduce your problem with a new user account instead of the current your account? (yes / no): haven't tried

**Comment by fujiwarat** [comment]
Seems this bug is a QT5 specific. GTK applications are no problem.
I will report the issue to QT later.

**Comment by rodrigograca31** [comment]
Yeah this seems to be specific on the application/OS side that doesnt have emoji support

**Comment by Iizuki** [comment]
Years later, I stumbled on this. Is there a QT upstream issue of this @fujiwarat  (and what might have become of it)?

**Comment by fujiwarat** [comment]
Thank you for your ping. I forgot this issue.
I no longer reproduce this issue in Qt 6.10.1 + Plasma 6.5.5 in Fedora 43 as I also observed the issue previously.
I don't know which is the QT upstream issue but hope it's resolved in the upstream.
